.github/workflows/reusable_citation_check.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Reusable Citation Check'
+
+on:
+  workflow_call:
+
+jobs:
+  citation_check:
+    name: 'Check Citation'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 'Check out the repo'
+        uses: actions/checkout@v4
+
+      - name: 'Setup Python'
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
```

Comment:
While we're at it maybe we upgrade to a more recent Python?

Reply:
Sure! 3.12 or something older? :)

Reply:
@jwallwork23 ping for notification :)

Reply:
> Sure! 3.12 or something older? :)

Yeah give that a go.

Reply:
> @jwallwork23 ping for notification :)

No need to do this any more btw - I turned on all notifications for the mesh-adaptation repos.

Reply:
Done in d4c059d1dfb3abdb24e2686b353598950bfe64d5